includes/classes/class-rtcamp-google-embeds.php

```diff
@@ -442,6 +444,32 @@ public function get_thumb_preview( \WP_REST_Request $request ) {
 		$data['preview_url'] = $this->get_thumbnail_url( $file_id );
 		return new \WP_REST_Response( $data, 200 );
 	}
+
+	/**
+	 * Save thumbnail of a doc.
+	 *
+	 * @param string $file_id File ID.
+	 * @param mixed  $contents Contents of file.
+	 *
+	 * @return void
+	 */
+	public function save_thumbnail( $file_id, $contents ) {
+		$uploaddir = wp_upload_dir();
+
+		$uploadpath = $uploaddir['basedir'] . '/cache/wp-google-drive/';
```

Comment:
Check if the `$uploaddir` is not empty or not getting any error.

Reply:
@dharmin, I think `wp_upload_dir()` will always return the upload path of WP. So here it's not necessary to check for empty of getting error.